starport-core/src/main/scala/com/krux/starport/CleanupExistingPipelines.scala

```diff
@@ -128,6 +128,10 @@ object CleanupExistingPipelines extends StarportActivity {
             logger.info(s"insert ${failedPipeline.awsId} to failed pipelines")
             db.run(DBIO.seq(FailedPipelines() += failedPipeline)).waitForResult
           }
+
+        PipelineHistoryHelper.updatePipelineHistories(healthyPipelines, HealthStatus.SUCCESS)
+        PipelineHistoryHelper.updatePipelineHistories(failedPipelines, HealthStatus.FAILED)
```

Comment:
These two steps need to happen before any pipelines are deleted or marked as not tracked in the DB, in case it fails the job can recover without missing any pipelines